js/index.js

```diff
@@ -22,5 +22,91 @@ for(i = 0; i < skills.length; i++) {
 
 console.log(skillsList);
 
-// console.log(thisYear);
 
+// v3 works
+const messageForm = document.querySelector('[name="leave_message"]');
+
+// add event listner on submit and capture 3 values in the form
+messageForm.addEventListener("submit", function(e) {
+  // console.log(e);
+  
+  e.preventDefault();
+  
+  // log the value of the form fields
+  nameValue = e.target.name.value;
+  console.log(nameValue);
+  emailValue = e.target.email.value;
+  console.log(emailValue);
+  const messageValue = e.target.message.value;
+  console.log(messageValue);
+
+  // validate forms
+  if(nameValue === ''  || emailValue === '' || messageValue === '') {
+    alert("Please fill out all fields");
+    return false      
+  };
+
+  const messageSection = document.querySelector('#messages');
+  const messageList = messageSection.querySelector('ul');
+
+  // hide messages by default
+  messageSection.style.display = 'none';
+
+ 
+  // show emailto link using name, and show the message
+  const newMessage = document.createElement('li');
+  newMessage.innerHTML=`<a style="color:orange;" href="mailto:${emailValue}">${nameValue}</a> <span> wrote: ${messageValue}</span>`; 
```

Comment:
In the future, you'll want to be careful when templating user input into a string that will be interpreted in some other language: It can make you vulnerable to code injection attacks.